commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -35,32 +38,53 @@ def run_filter(self, request):  # pylint: disable=arguments-differ
         Returns:
             dict:
                 active_order_management_system: result from pipeline steps to determine if
-                            redirect_to_commercetools_checkout flag is enabled and course_run_key
-                            and sku query params exist and detect which checkout to redirect to
+                            redirect_to_commercetools_checkout flag is enabled and course_run_key,
+                            sku, and bundle query params exist and detect which checkout to redirect to
         """
-        sku = request.query_params.get('sku', '').strip()
+        sku_params = request.query_params.getlist('sku')
         course_run = request.query_params.get('course_run_key', '').strip()
-        commercetools_available_course = None
+        bundle = request.query_params.get('bundle', '').strip()
+        commercetools_available_product = None
 
-        if course_run and is_redirect_to_commercetools_enabled_for_user(request):
+        if is_user_enterprise_learner(request):
+            return {
+                ACTIVE_ORDER_MANAGEMENT_SYSTEM_KEY: FRONTEND_APP_PAYMENT_CHECKOUT
+            }
+
+        ct_api_client = CommercetoolsAPIClient()
+
+        if bundle and is_program_redirection_to_ct_enabled(request):
+            try:
+                commercetools_available_product = ct_api_client.get_product_by_program_id(bundle)
+                if not commercetools_available_product:
+                    logger.warning(
+                        f'[get_product_by_program_id] Program {bundle} not found in Commercetools. '
+                        f'Please ensure it is properly synced.'
+                    )
+            except HTTPError as exc:
+                logger.exception(
+                    f'[get_product_by_program_id] Failed to get CT program '
+                    f'for product_id: {bundle} with exception: {exc}'
+                )
```

Comment:
This block is not covered in unit tests.

Reply:
added tests for pipeline